core/indexer/processTransactions.go

```diff
@@ -248,6 +251,33 @@ func (tdp *txDatabaseProcessor) groupNormalTxsAndRewards(
 	return transactions, rewardsTxs
 }
 
+func (tdp *txDatabaseProcessor) setTransactionSearchOrder(transactions map[string]*Transaction, shardId uint32) map[string]*Transaction {
+	currentOrder := 0
+	shardIdentifier := tdp.createShardIdentifier(shardId)
```

Comment:
shardID

Reply:
ShardID is the parameter passed to the function. I used "identifier" to: 1. avoid name collision and 2. to avoid confusion between an actual shard id in Elrond context and this custom identifier